### PR TITLE
shows interviewer domains in interviewer list

### LINKED ISSUE #84: Show interviewer domains in interviewers list
Currently, the list returned by the `.interviewers` command only shows an interviewer's discord name and their Calendly link. It'd be nice if we also included their domains as well so interviewees have a better idea of who to contact.

### PR DESCRIPTION
**Pull request by elainehan3**
# Related Issues
resolves #84

# Summary of Changes 
interviewer list now shows interviewer domains, if they have any

# Steps to Reproduce
run .interviewers

### CODE REVIEW

File: src/components/interview.ts [hunk 1]
```diff
@@ -103,6 +103,18 @@ export const getInterviewers = async (
   return res;
 };
 
+export const getInterviewerDomainsString = async (id: string): Promise<string> => {
+  const userDomains = await getDomains(id);
+  if (userDomains.length === 0) {
```
Comment (gzcharleszhang) [inline src/components/interview.ts:108]: Is it possible to reuse the `getDomainsString` helper? It's defined near the top of this file.

Comment (elainehan3) [reply src/components/interview.ts:108]: Yes, sorry about that, didn't notice the function, I've changed it.

Comment (gzcharleszhang) [reply src/components/interview.ts:108]: No need to apologize! I'll approve the PR once you push the new changes.

File: src/components/interview.ts [hunk 2]
```diff
@@ -103,6 +103,15 @@ export const getInterviewers = async (
...
```
Comment (gzcharleszhang) [inline src/components/interview.ts:108]: nit: you can simplify this with a ternary operator like so
```
return userDomains.length === 0 ? `` : getDomainsString(userDomains)
```

**Review by gzcharleszhang — APPROVED** [review]
1 minor comment, everything else looks good. Nice job!

File: src/components/interview.ts (hunk 2 above)
Comment (elainehan3) [reply src/components/interview.ts:108]: Ah ok, thanks for the tip, I've never used those before, but that definitely looks shorter, thanks!

**Pull request merged**